icons/solidjs/solidjs-original-wordmark.svg

```diff
@@ -0,0 +1,57 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 128 128">

```

Comment:
Please replace this icon withthe wordmark version from here:
https://www.solidjs.com/media

Notice how the text color and font is different